.github/workflows/workflow.yaml

```diff
@@ -317,6 +317,60 @@ jobs:
             done
           done
 
+  benchmark:
```

Comment:
how did you test this end to end? (i ask bc I am writing a GH wf too hehe)

Reply:
I tested this job on my fork by manually creating releases and generating needed artifacts. Then testing this job against two branches in my fork.

Reply:
i've been using `act` to test locally too. it helps w the development iteration: https://github.com/nektos/act